spec/features/check_users_spec.rb

```diff
@@ -23,21 +26,20 @@
     it "successful login" do
       create(:user)
       login("user@example.com", "password")
-      expect(page).to have_content "Добро пожаловать!"
+      expect(page).to have_content I18n.t(:welcome)
     end
 
     it "failed login" do
       create(:user)
       login("user_example.com", "password")
-      expect(page).to have_content "Неправильный логин или пароль!"
+      expect(page).to have_content I18n.t(:fl_data_err)
     end
 
     it "logout" do
       create(:user)
       login("user@example.com", "password")
-      click_link "Выйти"
-      expect(page).to have_content "Зарегистрируйтесь или войдите для \
-        начала работы с карточками"
+      click_link I18n.t('.layouts.application.logout')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.